### ISSUE TITLE
Nulecule

### ISSUE THREAD

**Issue by asamalik** [developer-portal/content#18]

**Comment by sferich888** [comment]
http://www.projectatomic.io/docs/nulecule/ provides a lot of what you need to get started, with nelecule, as well as https://github.com/projectatomic/nulecule/blob/master/docs/getting-started.md how/what is the best way to handle content that can/should be referenced from other locations/?

**Comment by phracek** [comment]
@sferich888 Yeah, we can use references, but basically some intro and even how to install it on Fedora would be useful. No complicated things.
